packages/java/parser-jvm-plugin-backbone/pom.xml

```
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.vaadin.fusion</groupId>
+    <artifactId>fusion-project</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
```

```suggestion
    <relativePath>../../../pom.xml</relativePath>
  </parent>
```

enables running `mvn install` once in the repository root to install all the packages.